<?xml version='1.0' encoding='UTF-8'?>
<org.jenkinsci.plugins.workflow.support.storage.SimpleXStreamFlowNodeStorage_-Tag plugin="workflow-support@2.16">
  <node class="org.jenkinsci.plugins.workflow.cps.nodes.StepAtomNode" plugin="workflow-cps@2.41">
    <parentIds>
      <string>18</string>
    </parentIds>
    <id>19</id>
    <descriptorId>org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep</descriptorId>
  </node>
  <actions>
    <org.jenkinsci.plugins.workflow.support.actions.LogActionImpl>
      <charset>windows-1252</charset>
    </org.jenkinsci.plugins.workflow.support.actions.LogActionImpl>
    <org.jenkinsci.plugins.workflow.cps.actions.ArgumentsActionImpl plugin="workflow-cps@2.41">
      <arguments>
        <entry>
          <string>script</string>
          <org.jenkinsci.plugins.workflow.actions.ArgumentsAction_-NotStoredReason>MASKED_VALUE</org.jenkinsci.plugins.workflow.actions.ArgumentsAction_-NotStoredReason>
        </entry>
      </arguments>
      <isUnmodifiedBySanitization>false</isUnmodifiedBySanitization>
    </org.jenkinsci.plugins.workflow.cps.actions.ArgumentsActionImpl>
    <org.jenkinsci.plugins.workflow.actions.TimingAction plugin="workflow-api@2.23.1">
      <startTime>1516083499644</startTime>
    </org.jenkinsci.plugins.workflow.actions.TimingAction>
    <org.jenkinsci.plugins.workflow.actions.ErrorAction plugin="workflow-api@2.23.1">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 1</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:335)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:289)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)</trace>
          <trace>java.util.concurrent.FutureTask.run(Unknown Source)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)</trace>
          <trace>java.lang.Thread.run(Unknown Source)</trace>
        </stackTrace>
        <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
          <c class="list"/>
          <list reference="../c"/>
        </suppressedExceptions>
      </error>
    </org.jenkinsci.plugins.workflow.actions.ErrorAction>
    <io.jenkins.blueocean.service.embedded.BlueOceanUrlAction plugin="blueocean-rest-impl@1.3.3">
      <blueOceanUrlObject class="io.jenkins.blueocean.service.embedded.BlueOceanUrlObjectImpl">
        <mappedUrl>blue/organizations/jenkins/pipelines/</mappedUrl>
        <modelObject class="org.jenkinsci.plugins.workflow.cps.nodes.StepAtomNode" reference="../../../../node"/>
      </blueOceanUrlObject>
    </io.jenkins.blueocean.service.embedded.BlueOceanUrlAction>
  </actions>
</org.jenkinsci.plugins.workflow.support.storage.SimpleXStreamFlowNodeStorage_-Tag>